book/src/rpc/mod.rs

```diff
@@ -1,2 +1,30 @@
+use futures::{future, prelude::*};
+use std::io;
+use std::net::SocketAddr;
+use tarpc::server::{BaseChannel, Channel, Handler};
+use tokio_serde::formats::Json;
+
+pub mod models;
 pub mod server;
 pub mod service;
+
+use crate::rpc::server::BookServer;
+use crate::rpc::service::BookService;
```

Comment:
Why are you not just `use server::BookServer` and `use service::BookService`? Is the Service import even needed here?

Reply:
The `struct BookServer` is imported in the line above and used in line 23. The `trait BookService` needs to be in scope to call the `.serve()` method in line 24.